app/models/user_coffee.rb

```diff
@@ -11,6 +11,10 @@ class UserCoffee < ApplicationRecord
   validates :weight_left, presence: true, numericality: { greater_than: -1 }
   accepts_nested_attributes_for :coffee
 
+  def name
+    coffee.name
+  end
```

Comment:
I don't understand the use of this method. Isn't coffee.name already enough ?

Reply:
Les user_coffee n'ayant pas de champs "name", ca permet de leur donner automatiquement un nom en fonction du Coffee auquel ils appartienent. C est utile dans la Form NEW extraction pour séléctionner un User_coffee.

Je commence donc peut-etre à entrevoir pourquoi tu voulais lié la table Extractions à la table Coffees et non pas User_Coffee, mais cela nous aurait probalement créer les problématique équivalentes avec les attribut d'un User_coffee et aurait compliqué l implémentation pour gérer le paramètre "Archvié" ou clacluler la quqnatité de café restante.... 🤷‍♂️

dans un second temps ca me fait penser que dans l éventualité (rare) où un user aurait plusieurs paquets de Farms Yungas en meme temps ( non archivés), ou bien du Farms Yunggas mais de 2 torrefacteurs différents, ou pas de la même année de torrefaction, ou des paquets de taille différentes.....
il faut que j'affiche d'autre info en plus du nom du café lorsque le USER slectionne un café dans le NEW d'une exctraction....